src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -573,6 +573,47 @@
         az acr task update-run -r MyRegistry --run-id runId --no-archive false
 """
 
+helps['acr task credential'] = """
+type: group
+short-summary: Credential for a task
+"""
+
+helps['acr task credential add'] = """
+type: command
+short-summary: Add a credential to the task
+examples:
+  - name: Enable the system-assigned credential to an existing task. This will replace all existing user-assigned identities.
+    text: >
+        az acr task credential add -n taskname -r registryname --login-server myregistry.docker.io --username 'myusername' --password 'mysecret'
+"""
+
+helps['acr task credential remove'] = """
+type: command
+short-summary: Remove credential for a task.
+examples:
+  - name: Remove a user-assigned credential from a task.
+    text: >
+        az acr task credential remove -n taskname -r registryname --login-server myregistry.docker.io
+"""
+
+helps['acr task credential update'] = """
+type: command
+short-summary: Update the credential for a task.
+examples:
+  - name: Update the credential for a task
+    text: >
+        az acr task credential update -n taskname -r registryname --login-server myregistry.docker.io --username 'myusername2' --password 'mysecret'
+"""
+
+helps['acr task credential show'] = """
+type: command
+short-summary: Display the Credentials for task.
+examples:
+  - name: Display the Credentials for a task.
+    text: >
+        az acr task credential show -n taskname -r registryname
```

Comment:
Does this show only registry credentials or entire credential property for a task?